package.json

```diff
@@ -30,28 +31,74 @@
     "url": "https://github.com/nextcloud/news/issues"
   },
   "engines": {
-    "node": "^14.0.0",
-    "npm": "^7.0.0"
+    "node": "^16.0.0",
+    "npm": "^8.0.0"
   },
   "resolutions": {
     "natives": "1.1.3"
   },
   "private": true,
   "homepage": "https://github.com/nextcloud/news",
   "devDependencies": {
+    "@babel/core": "^7.11.1",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.11.0",
     "@nextcloud/babel-config": "^1.0.0",
     "@nextcloud/browserslist-config": "^2.2.0",
-    "@nextcloud/eslint-config": "^8.0.0",
+    "@nextcloud/eslint-config": "^6.1.2",
+    "@nextcloud/eslint-plugin": "^2.0.0",
     "@nextcloud/stylelint-config": "^2.1.2",
-    "@nextcloud/webpack-vue-config": "^5.1.0"
+    "@nextcloud/webpack-vue-config": "^5.1.0",
+    "babel-loader": "^8.1.0",
+    "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-import-resolver-webpack": "^0.12.2",
+    "eslint-loader": "^4.0.2",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsdoc": "^37.9.7",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^5.2.0",
+    "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-vue": "^7.20.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "file-loader": "^6.0.0",
+    "jasmine-core": "^3.5.0",
+    "jquery": "^3.5.1",
+    "jshint": "^2.11.1",
+    "karma": "^5.0.9",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage": "^2.0.2",
+    "karma-firefox-launcher": "^1.3.0",
+    "karma-jasmine": "^2.0.1",
+    "linkifyjs": "^3.0.5",
+    "minimatch": "^3.0.4",
+    "natives": "^1.1.6",
+    "node-polyfill-webpack-plugin": "^1.1.4",
+    "node-sass": "^7.0.1",
+    "postcss": "^8.4.14",
+    "postcss-html": "^1.4.1",
+    "sass-loader": "^12.6.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^14.8.4",
+    "stylelint-config-recommended-scss": "^5.0.2",
+    "stylelint-config-recommended-vue": "^1.4.0",
+    "stylelint-webpack-plugin": "^2.1.0",
+    "url-loader": "^4.1.0",
+    "vue-loader": "^15.9.3",
+    "vue-template-compiler": "^2.6.11",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2"
   },
   "dependencies": {
     "@nextcloud/axios": "^1.10.0",
+    "@nextcloud/calendar-js": "^2.0.0",
```

Comment:
That doesn't seem correct

Reply:
looks like you're right, there is a 3.0.0 release, updated calendar-js

Reply:
I mean the fact that we depend on calendar components

Reply:
oh hmm. I can look into why I added this and see if it can be removed

Reply:
not sure why I added it, but I removed it and it still builds fine it seems